lib/filedialogs/portable-file-dialogs.h

```diff
@@ -1020,7 +1030,7 @@ inline internal::file_dialog::file_dialog(type in_type,
         OPENFILENAMEW ofn;
         memset(&ofn, 0, sizeof(ofn));
         ofn.lStructSize = sizeof(OPENFILENAMEW);
-        ofn.hwndOwner = GetForegroundWindow();
+        ofn.hwndOwner = GetActiveWindow();
```

Comment:
Hey, @pegvin Thanks for sending this pull request. Could you please use this commit's changes here: https://github.com/neutralinojs/neutralinojs/commit/d126777363102cd415ec1cd5cd9dec1b729df217

Perhaps, later we can introduce some patching way if our upstream libraries get modified frequently.

Thanks